SingularityService/src/main/java/com/hubspot/singularity/data/RequestManager.java

```diff
@@ -258,6 +256,14 @@ public SingularityCreateResult activate(SingularityRequest request, RequestHisto
     return save(request, RequestState.ACTIVE, historyType, timestamp, user, message);
   }
 
+  public SingularityCreateResult deleting(SingularityRequest request, RequestHistoryType historyType, long timestamp, Optional<String> user, Optional<String> message) {
+    return save(request, RequestState.DELETING, historyType, timestamp, user, message);
+  }
+
+  public SingularityCreateResult deleted(SingularityRequest request, RequestHistoryType historyType, long timestamp, Optional<String> user, Optional<String> message) {
```

Comment:
maybe for these methods we name them more like an action. i.e. `markDeleted` or `markDeleting`, just to clarify what they are doing. Especially since we already have a `deleteRequest` method (which could maybe be renamed to something like `startRequestDelete`?)